test/issue-636

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+AURDEST="$(mktemp -d --tmpdir "aurutils-${BASH_SOURCE[0]##*/}.XXXXXXXX")"
```

Comment:
I think it suffices to set `argv0=issue-636`, `aurutils-$argv0` as other `aur` scripts

Reply:
Done.